src/App.jsx

```diff
@@ -0,0 +1,111 @@
+import { useState, useEffect, Fragment } from "react";
+import api from "./api/todos";
+import ToDo1 from "./component/ToDo1";
+import ToDoItem from "./component/ToDoItem";
+
+import "./styles/style.css";
+
+function App() {
+  const [theme, setTheme] = useState("light");
+  const [tasks, setTasks] = useState([]);
+
+  //retrive Tasks
+  const retriveTasks = async () => {
+    const response = await api.get("/tasks");
+    return response.data;
+  };
```

Comment:
use arrow function over normal functions
Handle the error

Reply:
resolved